docs/pywbemcligeneraloptions.rst

```diff
@@ -117,7 +114,7 @@ PYWBEMCLI_NAME                  ``--name``
 PYWBEMCLI_USER                  ``--user``
```

Comment:
Instead of naming the heading of the first column "Export Name", change to "Environment Variable".

Reply:
DONE